crates/lsp/src/lib.rs

```diff
@@ -199,6 +199,9 @@ fn on_code_lens_request(
                 title: TEST_CODELENS_TITLE.into(),
                 command: TEST_COMMAND.into(),
                 arguments: Some(vec![
+                    // TODO: Change this if we change the flag
```

Comment:
How would we change this flag?

Reply:
I believe @kobyhallx wants to rename it to `--root` or `--root-dir` in the future. This is trying to have a reminder to change this. But maybe the TODO should be at the location of the clap macro instead